### PR TITLE
Sdk: include rfq protocol fees

### PR DESCRIPTION
**Pull request by ChiTimesChi**
**Description**
This PR adds support for protocol fees in the `FastBridge` contract. 

## Summary by CodeRabbit

- **New Features**
	- Introduced a feature to retrieve the protocol fee rate within the routing system.
	- Updated routing calculations to include protocol fee adjustments, ensuring accurate cost representation during transactions.

### CODE REVIEW

File: packages/sdk-router/src/rfq/fastBridgeRouterSet.ts [hunk 1]
```diff
@@ -112,13 +112,19 @@ export class FastBridgeRouterSet extends SynapseModuleSet {
...
       .map(({ quote, originQuery }) => ({
         quote,
         originQuery,
-        // Apply quote to the proceeds of the origin swap
+        // Apply quote to the proceeds of the origin swap with protocol fee applied
         // TODO: handle optional gas airdrop pricing
-        destAmountOut: applyQuote(quote, originQuery.minAmountOut),
+        destAmountOut: applyQuote(
+          quote,
+          this.applyProtocolFeeRate(originQuery.minAmountOut, protocolFeeRate)
```
Comment (mikeyrf) [inline packages/sdk-router/src/rfq/fastBridgeRouterSet.ts:126]: Applied on origin amount tho so is it relevant for FE as only used by caller of `claim()` which will be relayer?

**Review by mikeyrf — APPROVED** [review]
One question but otherwise lgtm

File: packages/sdk-router/src/rfq/fastBridgeRouterSet.ts (hunk 1 above)
Comment (mikeyrf) [reply packages/sdk-router/src/rfq/fastBridgeRouterSet.ts:126]: Other consideration would be on `refund()`, bridge sends back to origin sender `transaction.originAmount + transaction.originFeeAmount` so protocol doesn't take a fee on unsuccessful bridges, if that affects SDK in any way.

Comment (ChiTimesChi) [reply packages/sdk-router/src/rfq/fastBridgeRouterSet.ts:126]: I am replicating the internal logic of `FastBrdige.bridge()` wrt the origin amount used for the cross-chain swap:
https://github.com/synapsecns/sanguine/blob/1b95502e029e6e14b588b5dff8a9c3ad0d4c76ec/packages/contracts-rfq/contracts/FastBridge.sol#L99

This should be the amount that the off-chain Relayer will recognize, and therefore to get an accurate final quote for the user, the Relayer's quote is applied to the amount after the protocol fees.

Does this make sense?

Comment (mikeyrf) [reply packages/sdk-router/src/rfq/fastBridgeRouterSet.ts:126]: Yup! That makes sense then

**Pull request merged**